### ISSUE TITLE
PVR/VNSI issues

### ISSUE THREAD

**Issue by TheChief79** [CoreELEC/CoreELEC#10]
As i already wrote in LE-forum, me and friend having problems with LiveTV in 8.90.0. No audio and video stops after few seconds. We have both H96 Pro + with 912 SOC and use VDR/vnsiserver/VNSI Addon.

Logfiles attached:

[log-2018-04-16-07.17.15.zip](https://github.com/CoreELEC/CoreELEC/files/1917247/log-2018-04-16-07.17.15.zip)

Just opend that bug report again for better visibility and to track the status.

Thanks for your work.

**Comment by adamg88** [comment]
Have you tried TVH to see if it is just an add-on related issue rather than a build issue?

**Comment by TheChief79** [comment]
No, but i can try if i find some time. But i think it could be addon related, because dvbviewer addon seems to work for my friend. On the other hand, everything worked with vnsi until the new release.

**Comment by TheChief79** [comment]
TVH is working, audio and video.

**Comment by afl1** [comment]
I pushed fix for this issue. In next release it'll be fixed.
[https://github.com/CoreELEC/CoreELEC/commit/cc036516eef967cd992323e684f3b53bb027814f](url)

**Comment by TheChief79** [comment]
Cool thanks. I would build by myself, but it takes too long :D

**Comment by TheChief79** [comment]
Vnsi works with your fix but as i already told boot2k3, remote does not work die to latest kodi changes.

https://github.com/xbmc/xbmc/pull/13761

**Comment by adamg88** [comment]
Please keep all comments here so other developers know what you are referring to as myself and others are not privy to your conversation with boot2k3.

**Comment by TheChief79** [comment]
Ok, he just sent me a developer package with the fix from afl1:  https://github.com/CoreELEC/CoreELEC/commit/cc036516eef967cd992323e684f3b53bb027814f

With this fix vnsi works again.
